index.bs

```diff
@@ -692,15 +764,14 @@ Registry instances are initially created with the internal slots described in th
 
 <h3 id="loading-operations">Loading Operations</h3>
 
-<h4 id="request-fetch" aoid="RequestFetch">RequestFetch(loader, key)</h4>
+<h4 id="request-fetch" aoid="RequestFetch">RequestFetch(entry)</h4>
 
 <emu-alg>
-1. Let _entry_ be EnsureRegistered(_loader_, _key_).
 1. Let _fetchStageEntry_ be GetStage(_entry_, "fetch").
 1. If _fetchStageEntry_ is *undefined*, return a promise resolved with *undefined*.
-1. Let _hook_ be GetMethod(_loader_, @@fetch).
+1. Let _hook_ be GetMethod(_entry_.[[Loader]], @@fetch).
 1. // TODO: metadata object
-1. Let _p0_ be the result of promise-calling _hook_(_key_).
+1. Let _p0_ be the result of promise-calling _hook_(_entry_).
```

Comment:
OK so this is a major bug: now that there's no key, there's no way for a fetch hook to ever know what to fetch. :)

Now, granted, part of this whole PR is about the fact that there isn't a single unique key for an entry since it can be stored at multiple keys or in multiple registries. But somewhere along the line, we _have_ to tell the fetch hook what we're trying to fetch.